### DISCUSSION TITLE
How to make builds more memory efficient

### DISCUSSION THREAD

**Discussion by shalior** [sveltejs/kit#7989] [Q&A] [answered]
Hey!

I'm getting ` JavaScript heap out of memory` when trying to build svelte kit in a docker container. The available ram is limited (2GB) so I'm looking for some tips and recommendations or workarounds to make svelte kit and adapter node use less memory.

Is consuming 2GB of ram normal? or maybe i did something wrong?

any help is appreciated. thanks.

**Comment by benmccann** [comment, marked as answer]
This is a known issue in Vite / Rollup: https://github.com/vitejs/vite/issues/2433

There are some tips in the Rollup FAQ: https://rollupjs.org/guide/en/#error-javascript-heap-out-of-memory
